Source/EyesRelax/Settings.cs

```diff
@@ -2,8 +2,12 @@
 
 namespace EyesRelax
 {
-    public class Settings
+    public class Settings : DefaultSettings
     {
-        public double MainTimerTime { get; set; }
+        public int WorkingTime { get; set; } //in minutes 
+        public int SmallBrake { get; set; } //in minutes
+        public static sbyte LongBreakCounter { get; set; } //counts long breaks
+        public int LongBreak { get; set; } //in minutes
+        public bool OverallWorkingTime { get; set; } //in hours & minutes
```

Comment:
Also, we need to add additional bool property to represent if a user selected to show Overall time or no.